zappa_settings.json

```diff
@@ -0,0 +1,14 @@
+{
+    "dev": {
+        "app_function": "server.app",
```

Comment:
I'm a little confused about Zappa's position - is it used in the AWS deployment, in the development server, or both? Based on `requirements.txt` it appears to be replacing `Werkzeug` (serving assets?)? If it's used in both, what is it that leads to `.flaskenv` only being used in development?

Reply:
It's used only to put our code into AWS and configure AWS correctly. (It knows nothing about `.flaskenv`, that's purely a Flask thing.)

It's not replacing Werkzeug, but I removed Werkzeug from `requirements.txt` because Zappa requires an earlier version of Werkzeug than the one we were using. We don't have a direct dependency on Werkzeug - just an indirect dependency via Flask and Zappa - and I thought it best to remove all the indirect dependencies from `requirements.txt` and let pip pull in the versions specified by our dependencies' requirements.

Reply:
Ok, that's a non-standard approach for `requirements.txt`, but seems fine.